### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by wfz666** [zhengchen1999/DAT#69]
Hi, I am now using 4 A100 to train the model, but it is really slow(5 days) for batch size 32, so I would change the batch size to bigger and also the learning rate and milestone. So can you give me suggestions? Thanks!

**Comment by zhengchen1999** [comment]
Scale the milestone and learning rate proportionally with the batch size. Later, you can further adjust the lr according to the training effect.
